_config.yml

```diff
@@ -198,6 +198,24 @@ flink_kubernetes_operator_releases:
         asc_url: "https://downloads.apache.org/flink/flink-kubernetes-operator-0.1.0/flink-kubernetes-operator-0.1.0-helm.tgz.asc"
         sha512_url: "https://downloads.apache.org/flink/flink-kubernetes-operator-0.1.0/flink-kubernetes-operator-0.1.0-helm.tgz.sha512"
 
+flink_table_store_releases:
+  -
+    version_short: "0.1"
+    source_release:
+      name: "Apache Flink Table Store 0.1.0"
+      id: "010-table-store-download-source"
+      flink_version: "1.15.0"
+      url: "https://www.apache.org/dyn/closer.lua/flink/flink-table-store-0.1.0/flink-table-store-0.1.0-src.tgz"
+      asc_url: "https://downloads.apache.org/flink/flink-table-store-0.1.0/flink-table-store-0.1.0-src.tgz.asc"
+      sha512_url: "https://downloads.apache.org/flink/flink-table-store-0.1.0/flink-table-store-0.1.0-src.tgz.sha512"
+    binaries_release:
+      name: "Apache Flink Table Store Binaries 0.1.0"
+      id: "010-kubernetes-operator-download-binaries"
+      flink_version: "1.15.0"
+      url: "https://repo.maven.apache.org/maven2/org/apache/flink/flink-table-store-dist/0.1.0/flink-table-store-dist-0.1.0.jar"
+      asc_url: "https://repo.maven.apache.org/maven2/org/apache/flink/flink-table-store-dist/0.1.0/flink-table-store-dist-0.1.0.jar.asc"
+      sha1_url: "https://repo.maven.apache.org/maven2/org/apache/flink/flink-table-store-dist/0.1.0/flink-table-store-dist-0.1.0.jar.sha1"
```

Comment:
`sha1` or `sha512` ?  Why the dist artifiact has the different sha algorithm as the source artifiacts ?

Reply:
Because we are providing jar, jar only has sha1